### PR TITLE
Vdb 1861 clip kick

### PR DESCRIPTION
**Pull request by yaoandrew**

### CODE REVIEW

File: transformers/component_tests/initializers/events/event_initializers_test.go [hunk 1]
```diff
@@ -131,6 +132,14 @@ var _ = Describe("Event transformer initializers", func() {
 		assertCorrectInit(transformer, name, topic, 8928165, catContracts())
 	})
 
+	It("configures clip kick", func() {
+		transformer := clip_kick.EventTransformerInitializer(db)
+		name := "clip_kick"
+		topic := "0x898eb267e6ff99efebaf458079f659c812f378fe8cf2a33ab4028717672bd021"
```
Comment (ecmendenhall) [inline transformers/component_tests/initializers/events/event_initializers_test.go:140]: Just for my own understanding, which components here are real and which are fake? Looks like this is a real topic, discovered by running the contracts locally, but the address and block are faked?

**Review by ecmendenhall — APPROVED** [review]
📎 🦵 ⚽

File: transformers/component_tests/initializers/events/event_initializers_test.go (hunk 1 above)
Comment (yaoandrew) [reply transformers/component_tests/initializers/events/event_initializers_test.go:140]: That's right.. Those are to be replaced with real values when available on `mainnet`.  I was trying to make them look really fake 🤣

File: db/migrations/20210318214325_create_clip_kick.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+(
+    id         SERIAL PRIMARY KEY,
+    header_id  INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
+    address_id BIGINT  NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    log_id     BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
+    bid_id     NUMERIC NOT NULL,
+    top        NUMERIC,
+    tab        NUMERIC,
+    lot        NUMERIC,
+    usr        TEXT,
```
Comment (yaoandrew) [inline db/migrations/20210318214325_create_clip_kick.sql]: `usr` and `kpr` should probably be `BIGINT` here in keeping with the convention of having addresses stored as IDs.

**Pull request merged**